backend/group.py

```diff
@@ -9,24 +9,27 @@
 
 class Group(Resource):
     def get(self, group_id):
+        return self.getObj(group_id).to_json
+
+    def getObj(self, group_id):
         try:
-            return Groups.objects.get(id=group_id)
+            return Groups.objects.get(id=group_id).to_json()
         except (DoesNotExist):
             raise DoesNotExist
 
     @jwt_required()
     def put(self, group_id):
         admin_id = get_jwt_identity()
         User.checkIfAdmin(self, admin_id)

```

Comment:
I think `checkIfAdmin` could also be included in the `checkIfRole` method instead of being separate.

Reply:
fixed